### PR TITLE
[11.0][IMP] stock_orderpoint_mrp_link

### PR DESCRIPTION
**Pull request by HviorForgeFlow**
Improve usability:
- [x] Add Purchase like right-top button for MO on Orderpoint views.

https://github.com/OCA/stock-logistics-warehouse/pull/579

@Eficent

### CODE REVIEW

File: stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py [hunk 1]
```diff
@@ -104,3 +104,6 @@ def test_02_stock_orderpoint_mrp_link_indirect_routing(self):
         self.assertTrue(mo2)
         self.assertEqual(mo2.orderpoint_id, self.orderpoint_secondary_loc)
         self.assertEqual(mo2.orderpoint_id, self.orderpoint_secondary_loc)
+
+    def test_03_stock_orderpoint_mrp_link_action_view(self):
+        self.orderpoint_secondary_loc.action_view_mrp_productions()
```
Comment (etobella) [inline stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py]: The test could more extended? Maybe check that if you use the domain returned and search, the result is the expected?

**Review by etobella — APPROVED** [review]
Just a comment on the test, just testing the function is not the best option, check that the result is the expected somehow

File: stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py (hunk 1 above)
Comment (HviorForgeFlow) [reply stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py]: @etobella I tried something...

Comment (etobella) [reply stock_orderpoint_mrp_link/tests/test_stock_orderpoint_mrp_link.py]: Perfectly fine!

**Review by LoisRForgeFlow — COMMENTED** [review]
Looks good, some small comments:

1. Check travis
2. Could you change the button icon.
image

it is used already for purchases, and we also add it in DDMPR :sweat_smile:

**Comment by HviorForgeFlow** [comment]
@lreficent comments attendend

**Review by LoisRForgeFlow — APPROVED** [review]
Runbot error is unrelated. :+1:

**Pull request merged**